wordpress-mobile/gutenberg-mobile
# Merge Master (1.16.0) back to develop

Merges release 1.16.0 from master to develop

To test:
 - Smoke test the demo app just to see if all is correct.

Update release notes:

- [x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
